src/components/forms/PlanForm/index.tsx

```diff
@@ -244,6 +248,15 @@ const PlanForm = (props: PlanFormProps) => {
     return <Redirect to={redirectAfterAction} />;
   }
 
+  const onSubmitSuccess = (setSubmitting: any, payload: PlanDefinition) => {
```

Comment:
@kelvin-muchiri  can we move this to a util file. change any to a setState type its usually sth like `React.Dispatch<React.setStateAction<boolean>>`, ooh and a docstring would be nice